cpp/unreal_plugins/SimulationController/Source/SimulationController/SimulationController.cpp

```diff
@@ -290,9 +287,7 @@ void SimulationController::bindFunctionsToRpcServer()
         FGenericPlatformMisc::RequestExit(immediate_shutdown);
     });
 
-    rpc_server_->bindAsync("ping", []() -> std::string {
-        return "SimulationController received a call to ping()...";
-    });
+    rpc_server_->bindAsync("ping", []() -> std::string { return "SimulationController received a call to ping()..."; });
```

Comment:
I like the old syntax because it's more consistent with the other lambdas in this code.

Reply:
Can you please restore the old formatting for consistency with the other lambdas in this part of the code.